app/services/nasa_service.rb

```diff
@@ -0,0 +1,18 @@
+class NasaService
```

Comment:
Created initialize of api key, only tested existance, will make the actual call on this branch